tokio-sync/src/mpsc/rx.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Sorry for being that late, but what was the reason for this file and `tx.rs`, if both are empty and not used?

Reply:
Ah... just forgot to delete them I guess.